install/docker/Dockerfile

```diff
@@ -1,4 +1,10 @@
 FROM docker:dind
+ 
+#install go
+COPY --from=golang:1.13-alpine /usr/local/go/ /usr/local/go/
```

Comment:
Does Golang:1.15-alpine work? We want to eventually move to go 1.16. And if 1.15 works,  it would  be in the right direction. Rest of the PR looks great.

Reply:
Done